packages/docs/command-line-interface/account.md

```diff
@@ -46,6 +160,59 @@ EXAMPLE
 
 _See code: [packages/cli/src/commands/account/new.ts](https://github.com/celo-org/celo-monorepo/tree/master/packages/cli/src/commands/account/new.ts)_
 
+### Register
+
+Register an account
+
+```
+USAGE
+  $ celocli account:register
+
+OPTIONS
+  --from=0xc1912fEE45d61C87Cc5EA59DaE31190FFFFf232d  (required) Account Address
+  --name=name                                        (required)
+
+EXAMPLE
+  register
+```
+
+_See code: [packages/cli/src/commands/account/register.ts](https://github.com/celo-org/celo-monorepo/tree/master/packages/cli/src/commands/account/register.ts)_
+
+### Register-metadata
+
+Register metadata about an address
+
+```
+USAGE
+  $ celocli account:register-metadata
+
+OPTIONS
+  --from=0xc1912fEE45d61C87Cc5EA59DaE31190FFFFf232d  (required) Addess of the account to set metadata for
+  --url=htttps://www.celo.org                        (required) The url to the metadata you want to register
+
+EXAMPLE
+  register-metadata --url https://www.celo.org --from 0x0
+```
+
+_See code: [packages/cli/src/commands/account/register-metadata.ts](https://github.com/celo-org/celo-monorepo/tree/master/packages/cli/src/commands/account/register-metadata.ts)_
+
+### Show-metadata
```

Comment:
What is the difference between this and get-metadata?

Reply:
`show-metadata` displays the contents of a local file. It would be useful to iteratively check that the metadata that you are about to upload and register is what you intend to register
`get-metadata` looks up the URL of the metadata file on the contract, fetches it, and then displays the contents of the file.

Reply:
Just a nit, the way we typically use `show` elsewhere in the CLI is more similar to the way you use `get` here